test/utils.sh

```diff
@@ -160,26 +166,30 @@ function deploy_test_env {
   export CONJUR_APPLIANCE_URL=$conjur_appliance_url
   export CONJUR_AUTHN_URL=$conjur_authenticator_url
 
-   echo "Deploying test-env"
-   $TEST_CASES_K8S_CONFIG_DIR/test-env.sh.yml | $cli create -f -
+  echo "Deploying test-env"
+  $TEST_CASES_DIR/test-env.sh.yml | $cli create -f -
 
-   expected_num_replicas=`$TEST_CASES_K8S_CONFIG_DIR/test-env.sh.yml |  awk '/replicas:/ {print $2}' `
+  expected_num_replicas=`$TEST_CASES_DIR/test-env.sh.yml |  awk '/replicas:/ {print $2}' `
 
-   # deploying deploymentconfig might fail on error flows, even before creating the pods. If so, retry deploy again
-   wait_for_it 600 "$cli get dc/test-env -o jsonpath={.status.replicas} | grep '^${expected_num_replicas}$'|| oc rollout latest dc/test-env"
+  # deploying deploymentconfig might fail on error flows, even before creating the pods. If so, retry deploy again
```

Comment:
this comment is a bit misleading as in k8s we have a `deployment` and not a `deploymentconfig`